package.json

```diff
@@ -21,8 +21,8 @@
   },
   "scripts": {
     "lint": "eslint --ignore-path .gitignore .",
-    "test": "nightwatch",
-    "test:coverage": "nyc yarn test",
+    "test": "MOZ_HEADLESS=1 nightwatch",
+    "test:coverage": "MOZ_HEADLESS=1 nyc yarn test",
```

Comment:
Shouldn't need this second one since it calls the other one.

Also, do we want to use cross-env to set this or meh?

Reply:
Dammit, you're right. Lol moving too fast